app/views/api/v2/domains/main.json.rabl

```diff
@@ -3,3 +3,9 @@ object @domain
 extends "api/v2/domains/base"
 
 attributes :fullname, :dns_id, :created_at, :updated_at
+
+@object.smart_proxies.keys.each do |proxy|
```

Comment:
I'd suggest putting this into a partial like api/v2/taxonomies/children_nodes, it's identical in every location.